File: server/src/main/java/com/ps/server/dto/CreateGameDTO.java
package com.ps.server.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class CreateGameDTO {

    private Long firstPlayerId;

    private Long secondPlayerId;

    public Long getFirstPlayerId() {
        return firstPlayerId;
    }

    public Long getSecondPlayerId() {
        return secondPlayerId;
    }

    public void setFirstPlayerId(Long firstPlayerId) {
        this.firstPlayerId = firstPlayerId;
    }

    public void setSecondPlayerId(Long secondPlayerId) {
        this.secondPlayerId = secondPlayerId;
    }
}

File: client/src/app/DTO/createGameDTO.ts

/** @ignore*/
export class CreateGameDTO {
    /** */
    public firstPlayerId: number;
    /** */
    public secondPlayerId: number;
    /** */
    public isRanked: boolean;


    /** @ignore*/
    constructor(firstPlayerId: number, secondPlayerId: number, isRanked: boolean) {
        this.firstPlayerId = firstPlayerId;
        this.secondPlayerId = secondPlayerId;
        this.isRanked = isRanked;
    }
}